### PR TITLE
Bugfix/add workplace success page links

### PR DESCRIPTION
**Pull request by omarCreativeDev**
- store newly created workplace uid on service so that complete page can route properly to create user, add staff records and workplace information pages

### CODE REVIEW

**Comment by omarCreativeDev** [comment]
[Add a subsidiary workplace - success page links](https://trello.com/c/7ezgufVx/299-add-a-subsidiary-workplace-success-page-links)

**Comment by tgreyuk** [comment]
[WDF Report naming incorrect](https://trello.com/c/nGpYMocV/309-wdf-report-naming-incorrect)

File: src/app/core/services/workplace.service.ts [hunk 1]
```diff
@@ -11,9 +11,10 @@ import { ErrorDefinition } from '@core/model/errorSummary.model';
 })
 export class WorkplaceService {
   constructor(private http: HttpClient) {}
-  public addWorkplaceInProgress$: BehaviorSubject<boolean> = new BehaviorSubject(false);
   public addWorkplaceFlow$: BehaviorSubject<string> = new BehaviorSubject(null);
+  public addWorkplaceInProgress$: BehaviorSubject<boolean> = new BehaviorSubject(false);
   public locationAddresses$: BehaviorSubject<Array<LocationAddress>> = new BehaviorSubject(null);
+  public newWorkplaceUid$: BehaviorSubject<string> = new BehaviorSubject(null);
```
Comment (ChazUK) [inline src/app/core/services/workplace.service.ts]: does this need to be an observable?

Comment (ChazUK) [reply src/app/core/services/workplace.service.ts]: BehaviourSubject*

**Pull request merged**